services/madoc-ts/src/frontend/site/features/EmbedItem.tsx

```diff
@@ -0,0 +1,37 @@
+import { blockEditorFor } from '../../../extensions/page-blocks/block-editor-for';
+import React from 'react';
+import styled from 'styled-components';
+
+const EmbedWrapper = styled.div`
+  iframe {
+    border: none;
+  }
+`;
+export const EmbedItem: React.FC<{
+  link?: string;
+  height?: string;
+  width?: string;
+}> = ({ link, height, width }) => {
+  return (
+    <EmbedWrapper>
+      <iframe src={link} height={height} width={width} />
```

Comment:
For security we should add some extra properties to limit what the iframe has access to.
- [`sandbox=""`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox)
- [`referrerpolicy="no-referrer"`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-referrerpolicy)

There may be others we could add, but I think limiting what the frame can do would be enough for now. We could add options to the block later too.

Reply:
`sandbox="" `dosent allow any scripts so most embeded widgets wont do much I added` sandbox="allow-scripts" `instead?

Reply:
I can remove this block if theres no usecase for it?

Reply:
I think it still might be useful!